### DISCUSSION TITLE
Pylenium and Pycharm

### DISCUSSION THREAD

**Discussion by dhurtado07** [ElSnoMan/pyleniumio#261] [General]
I have a problem when I want to run the following test from pycharm
1. I create a new project in pycharm 
2. I install pylenium 
`pip install pyleniumio`
3. start pylenium 
`pylenium init`
4. I create a file with the following
`def test_google_search (py):
	py.visit('https://www.google.com')
	py.get("[name='q']").type('puppies')
	py.get("[name='btnK']").submit()
	assert py.should().contain_title('puppies')`
5. I have configured by default the test runner as pytest
6.  when I run the test I get the following error 
![image](https://user-images.githubusercontent.com/9272385/169449385-8e49d889-97fa-48cd-b5ca-bfa270d283e9.png)

**Comment by ElSnoMan** [comment]
It doesn't see that requests is installed even though installing pylenium includes that automatically. That suggests either an error in installation or configuration. Can you share more of your code, project structure, and how you're using your virtual environment?

**Reply by dhurtado07** [reply]
I solved the problem, I installed the package from the Python packages tab, thank you 
![image](https://user-images.githubusercontent.com/9272385/169452088-b7521752-6741-4241-97d2-86df9677b9cf.png)

**Reply by ElSnoMan** [reply]
Excellent! For anyone else seeing this, PyCharm creates a virtual environment for you automatically so you have to use that one. Installing from Python Packages will use that virtual environment in case you don't not how to work with them directly
